README.md

```diff
@@ -137,7 +137,9 @@ Experimental feature, not production ready, waiting for the official standard fo
 
 The keys are derived according to [FLIP-200](https://github.com/onflow/flow/pull/200)
 
-- get `seed` from web3auth
+- get `entropy` from web3auth
+- create `mnemonic` from `entropy` using bip39
```

Comment:
I'd add `this ensures portability of the account to NuFi wallet even without account linking` (with "NuFi" and "account linking" being links)